internal/endtoend/testdata/inflection/postgresql/pgx/v4/go/models.go

```diff
@@ -4,6 +4,10 @@
 
 package querytest
 
+type Bonuse struct {
+	ID string
+}
```

Comment:
This appears to still be wrong here?

Reply:
yep, I expected "bonuses" table become "Bonus" after generated instead of "Bonuse"

Reply:
Yep, I want the "bonuses" table to become "Bonus" instead of "Bonuse" after generate

Reply:
I just deleted this one from PR 👍